tests/format/angular/angular/jsfmt.spec.js

```diff
@@ -1,5 +1,5 @@
-run_spec(__dirname, ["angular"], { trailingComma: "none" });
-run_spec(__dirname, ["angular"]);
-run_spec(__dirname, ["angular"], { printWidth: 1 });
-run_spec(__dirname, ["angular"], { htmlWhitespaceSensitivity: "ignore" });
-run_spec(__dirname, ["angular"], { bracketSpacing: false });
+run_spec(import.meta, ["angular"], { trailingComma: "none" });
```

Comment:
Why `run_spec` receive `import.meta`? Isn't `import.meta.url` is enough?

Reply:
`import.meta` might get useful properties we might want to use in the future, and it's shorter.

Reply:
I got it.